.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Experimental Feature Requirements
+about: Requirements for experimental features
+---
+
+This page lists the requirements for experimental features. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+- [ ] Feature is reviewed in a TOC meeting
```

Comment:
This seems a pretty high bar for experimental... maybe WG?

Reply:
+1 WG is probably good.  TOC for FYI.